genesis/src/main.rs

```diff
@@ -366,13 +362,25 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     poh_config.target_tick_duration =
         Duration::from_millis(value_t_or_exit!(matches, "target_tick_duration", u64));
 
+    let operating_mode = if matches.is_present("development") {
+        OperatingMode::Development
+    } else {
+        OperatingMode::SoftLaunch
+    };
+
     match matches.value_of("hashes_per_tick").unwrap() {
-        "auto" => {
-            let hashes_per_tick =
-                compute_hashes_per_tick(poh_config.target_tick_duration, 1_000_000);
-            println!("Hashes per tick: {}", hashes_per_tick);
-            poh_config.hashes_per_tick = Some(hashes_per_tick);
-        }
+        "auto" => match operating_mode {
+            OperatingMode::Development => {
+                let hashes_per_tick =
+                    compute_hashes_per_tick(poh_config.target_tick_duration, 1_000_000);
+                println!("Hashes per tick: {}", hashes_per_tick);
+                poh_config.hashes_per_tick = Some(hashes_per_tick);
+            }
+            _ => {
```

Comment:
I'd rather that `OperatingMode::SoftLaunch` be called out explicitly, as I'd *want* to have to consider all the matches if/when we add a new operating mode